README.md

```diff
@@ -1,22 +1,16 @@
 PyBitmessage
-============
+-----------
 
-Bitmessage is a P2P communications protocol used to send encrypted messages to
-another person or to many subscribers. It is decentralized and trustless,
-meaning that you need-not inherently trust any entities like root certificate
-authorities. It uses strong authentication, which means that the sender of a
-message cannot be spoofed, and it aims to hide "non-content" data, like the
-sender and receiver of messages, from passive eavesdroppers like those running
-warrantless wiretapping programs.
+Bitmessage is a peer-to-peer communication protocol used to send encrypted messages to another single person or to multiple subscribers. It is decentralized and trustless, meaning that one needs not inherently trust any entities such as root certificate authorities. Bitmessage uses strong authentication, meaning that the sender of a message cannot be spoofed. It aims to hide meta-data, such as the ID of sender and receiver of messages from passive eavesdroppers like those warrantless wiretapping programs which are ongoing. 
 
 
 Development
 ----------
-Bitmessage is a collaborative project. You are welcome to submit pull requests 
-although if you plan to put a non-trivial amount of work into coding new
-features, it is recommended that you first solicit feedback on the DevTalk
-pseudo-mailing list:
-BM-2D9QKN4teYRvoq2fyzpiftPh9WP9qggtzh
+
+Bitmessage is a collaborative project. You are welcome to submit pull requests - although if you plan to put a non-trivial amount of work into coding new features, it is recommended that you first solicit feedback on the DevTalk pseudo-mailing-list ("subscribe"): http://bm6hsivrmdnxmw2f.onion/chan/DevTalk 
```

Comment:
The DevTalk is gone. You can remove references to it. You can add reference to the contribution guidelines though (you can just copy&paste it from the wiki)